packages/manager/modules/pci/src/projects/project/data-processing/notebooks/terminate-notebook/terminate-notebook.controller.js

```diff
@@ -0,0 +1,22 @@
+export default class {
+  /* @ngInject */
+  constructor($state, dataProcessingService) {
+    this.$state = $state;
```

Comment:
Do you need to import `$state` ? I don't see any usage in the controller

Reply:
Removed :)